simulator/src/main/java/com/google/cloud/App.java

```diff
@@ -68,23 +78,23 @@ public class App {
   public static void main(String argv[]) {
 
     // Initialize the parsing objects
-    App app = new App();
+    App args = new App();
     JCommander parser = JCommander
         .newBuilder()
-        .addObject(app)
+        .addObject(args)
         .build();
 
     try {
 
       // Parse the user-specifiedc arguments
       parser.parse(argv);
-      if (app.help) {
+      if (args.help) {
         parser.usage();
         System.exit(1);
       }
 
       // Run the simulator
-      app.runSimulator();
+      args.runSimulator();
```

Comment:
Looks strange to ask arguments to run something. Perhaps renaming App to SimulatorRunner will be a reasonable naming change. Then "runner.runSimulator" will look pretty logical. Don't have to do it, just a suggestion.

Reply:
Agreed -- reverting this back. Not sure why I ended up doing that.